### PR TITLE
[9.0][MIG] Port l10n_nl_postcodeapi

### PR DESCRIPTION
**Pull request by astirpe**
It seems this was not ported yet...

### CODE REVIEW

**Comment by astirpe** [comment]
Here runbot will be green after `partner_street_number` is merged for Odoo9 (PR: https://github.com/OCA/partner-contact/pull/207)

**Review by hbrunn — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]

**Comment by hbrunn** [comment]
unfortunately, we can't merge until the dependency is merged, and this hangs on a comment :-(

**Comment by hbrunn** [comment]
@astirpe I can merge this after a rebase

**Pull request merged**
